python/ray/tune/tests/test_remote.py

```diff
@@ -0,0 +1,42 @@
+import unittest
+
+import ray
+from ray.tune import register_trainable, run_experiments, run
+from ray.tune.result import TIMESTEPS_TOTAL
+from ray.tune.experiment import Experiment
+from ray.tune.trial import Trial
+
+
+class RemoteTest(unittest.TestCase):
+    def tearDown(self):
+        ray.shutdown()
```

Comment:
Could you add a small test for checking that Tune works against the ray client?

Reply:
Done

Reply:
Added a pair